patient-service/src/main/java/com/raf/si/patientservice/controller/SchedMedExaminationController.java

```diff
@@ -26,21 +32,36 @@ public SchedMedExaminationController(SchedMedExaminationService schedMedExaminat
 
     @PreAuthorize("hasRole('ROLE_MED_SESTRA') or hasRole('ROLE_VISA_MED_SESTRA')")
     @PostMapping("/create")
-    public ResponseEntity<?> createSchedMedExamination(@Valid @RequestBody SchedMedExamRequest schedMedExamRequest){
+    public ResponseEntity<?> createSchedMedExamination(@Valid @RequestBody SchedMedExamRequest schedMedExamRequest) {
         return ResponseEntity.ok(schedMedExaminationService.createSchedMedExamination(schedMedExamRequest));
     }
 
     @PreAuthorize("hasRole('ROLE_DR_SPEC_ODELJENJA')" +
             "or hasRole('ROLE_DR_SPEC')" +
             "or hasRole('ROLE_DR_SPEC_POV')")
     @PutMapping("/update-exam-status")
-    public ResponseEntity<SchedMedExamResponse> updateSchedMedExaminationStatus(@Valid @RequestBody UpdateSchedMedExamRequest updateSchedMedExamRequest){
+    public ResponseEntity<SchedMedExamResponse> updateSchedMedExaminationStatus(@Valid @RequestBody UpdateSchedMedExamRequest updateSchedMedExamRequest) {
         return ResponseEntity.ok(schedMedExaminationService.updateSchedMedExaminationExamStatus(updateSchedMedExamRequest));
     }
 
+
+    @PreAuthorize("hasRole('ROLE_DR_SPEC_ODELJENJA')" +
+            "or hasRole('ROLE_DR_SPEC')" +
+            "or hasRole('ROLE_DR_SPEC_POV')" +
+            "or hasRole('ROLE_MED_SESTRA')" +
+            "or hasRole('ROLE_VISA_MED_SESTRA')")
+    @GetMapping("/search")
+    public ResponseEntity<Page<SchedMedExamResponse>> getSchedMedExam(@RequestHeader("Authorization") String authorizationHeader
```

Comment:
Nemoj da vracas page kao response. Pogledaj kako sam ja vracao npr. istoriju pregleda. Trebalo bi da imas listu SchedMedExamResponse-ova, i da ubacis count, odnosno koliko elemenata ima u bazi (to isto mozes da vidis u HealthRecordMapper-u).

Reply:
Igor me zamolio da vracam kao Page objekat zbog puginacije na frontu. Nije problem izmenicu.

Reply:
Svuda smo vracali kao listu i count, to je sve sto je potrebno na frontu. Nije im potreban Page objekat za paginaciju

Reply:
Nisam video mb, radim na izmeni